src/main/database.ts

```diff
@@ -0,0 +1,31 @@
+import {CGDatabase} from "../renderer/services/db/api";
+// @ts-ignore
+import {LevelDbController} from "../renderer/services/db/controller";
+import {config} from "@chainsafe/eth2.0-config/lib/presets/minimal";
+import {IService} from "../renderer/services/interfaces";
+import {CGAccount} from "../renderer/models/account";
+
+export class DatabaseHandler implements IService {
+    private database: CGDatabase;
+
+    public constructor() {
+        this.database = new CGDatabase({config, controller: new LevelDbController({name: "test-level.db"})});
```

Comment:
introduce config object as constructor param with default values.
I would suggest having:
- name - database name
- controller - so you can stub it out in tests

Reply:
Should I have a name alongside controller (because controller already includes name)?

Reply:
well you need  name here so you can pass it to controller. Not sure if I understood your question